scripts/gitlab_build.sh

```diff
@@ -9,5 +9,5 @@ rsync -rlv $2 $3 # $CHANNEL_DIR $BUILD_DIR
 
 ./bin/recipebook --changes origin/$1 recipes | ./bin/build \
  --recipes-dir $CI_PROJECT_DIR --artefacts-dir $3 --conda-build-image $CONDA_IMAGE \
- --build-channel $WSI_CONDA_CHANNEL --conda-uid 1001 --conda-gid 32001 \
- --verbose --remove-container
+ --build-channel $4 --conda-uid 1001 --conda-gid 32001 \
+ --verbose --remove-container #$WSI_CONDA_CHANNEL
```

Comment:
Maybe remove commented variable

Reply:
Happy to merge, though

Reply:
The comments are to show what the numbered command line options are.  They are present in all of the scripts, but if there is a better (or standard) way of documenting that, I can make general change.

Reply:
Heh. That was something I was going to open as a new (minor) issue - assigning the positional arguments to named variables, as you suggest. Meanwhile let's keep the comment.